### PR TITLE
Add multiple env vars per line

### LINKED ISSUE #1384: Env variables substitution in yaml config supports does not support more than one variable
In:
```
some_endpoint:
   url: ${VAR1} ${VAR2}
```
Only `${VAR1}` is substituted

In:
```
some_endpoint:
   url: "${VAR1} ${VAR2}" # with quotes
```
Nothing is substituted

https://github.com/RasaHQ/rasa_core/blob/8b5f9dd7c858b31d8d8e7fed1cd00df4fe37f74d/rasa_core/utils.py#L302

### PR DESCRIPTION
**Pull request by scottire**
**Proposed changes**:
- Closes #1384

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [x] added some tests for the functionality
- [x] updated the changelog

### CODE REVIEW

**Comment by akelad** [comment]
Thanks for submitting this PR, we'll give it a review as soon as possible

File: rasa_core/utils.py [hunk 1]
```diff
@@ -310,8 +310,12 @@ def replace_environment_variables():
     def env_var_constructor(loader, node):
         """Process environment variables found in the YAML."""
         value = loader.construct_scalar(node)
-        prefix, env_var, remaining_path = env_var_pattern.match(value).groups()
-        return prefix + os.environ[env_var] + remaining_path
+        expanded_vars = os.path.expandvars(value)
+        if '$' in expanded_vars:
+            raise KeyError(
+                "Environment variable {} does not exist".
```
Comment (wochinge) [inline rasa_core/utils.py]:
```suggestion
                "Error when trying to expand the environment variables in '{}'. "
                "Please make sure to also set these environment variables: '{}'.".
```

File: rasa_core/utils.py [hunk 2]
```diff
@@ -310,8 +310,12 @@ def replace_environment_variables():
     def env_var_constructor(loader, node):
         """Process environment variables found in the YAML."""
         value = loader.construct_scalar(node)
-        prefix, env_var, remaining_path = env_var_pattern.match(value).groups()
-        return prefix + os.environ[env_var] + remaining_path
+        expanded_vars = os.path.expandvars(value)
+        if '$' in expanded_vars:
+            raise KeyError(
```
Comment (wochinge) [inline rasa_core/utils.py]:
```suggestion
            not_expanded = [w for w in expanded_vars.split() if '$' in w]
            raise ValueError(
```

File: rasa_core/utils.py [hunk 3]
```diff
@@ -310,8 +310,12 @@ def replace_environment_variables():
...
+                "Environment variable {} does not exist".
+                format(expanded_vars))
```
Comment (wochinge) [inline rasa_core/utils.py]:
```suggestion
                format(value, not_expanded))
```

File: rasa_core/utils.py [hunk 4]
```diff
@@ -310,8 +310,14 @@ def replace_environment_variables():
...
         value = loader.construct_scalar(node)
-        prefix, env_var, remaining_path = env_var_pattern.match(value).groups()
-        return prefix + os.environ[env_var] + remaining_path
+        expanded_vars = os.path.expandvars(value)
+        if '$' in expanded_vars:
+            not_expanded = [w for w in expanded_vars.split() if '$' in w]
+            raise ValueError(
+                "Error when trying to expand the environment variables"
+                " in '{}'. Please make sure to also set these environment"
+                " variables: '{}'.".format(not_expanded, value))
```
Comment (wochinge) [inline rasa_core/utils.py]:
```suggestion
                " variables: '{}'.".format(value, not_expanded))
```

Comment (wochinge) [reply rasa_core/utils.py]: Sorry if I had a wrong order in the suggestion yesterday!

File: tests/test_utils.py [hunk 5]
```diff
@@ -174,5 +183,23 @@ def test_read_yaml_string_with_env_var_not_exist():
     user: ${USER_NAME}
     password: ${PASSWORD}
     """
-    with pytest.raises(KeyError):
+    with pytest.raises(ValueError) as error_info:
         r = utils.read_yaml_string(config_with_env_var_not_exist)
+    error_message = ("Error when trying to expand the environment variables "
```
Comment (wochinge) [inline tests/test_utils.py]: I think it's ok when we simply test the exception. Testing for the actual log message seems a bit too much for me.

File: tests/test_utils.py [hunk 6]
```diff
@@ -174,5 +183,23 @@ def test_read_yaml_string_with_env_var_not_exist():
...
+                     "in '['${PASSWORD}']'. Please make sure to also set these"
+                     " environment variables: '${PASSWORD}'.")
+    assert error_message in str(error_info.value)
+
+
+def test_read_yaml_string_with_multiple_env_vars_not_exist():
```
Comment (wochinge) [inline tests/test_utils.py]: As said above, I think it's not necessary to assert the actual error message. So I think we can skip this test.

**Comment by wochinge** [comment]
@scottire How is it going with the changes? Can I help you?

**Comment by scottire** [comment]
@wochinge I was under the impression off the changes were added.

**Comment by wochinge** [comment]
@scottire Sorry, saw it now :+1: Will merge as soon as travis is finished

**Comment by wochinge** [comment]
@scottire Merged 🚀 Thanks for your contribution!

**Pull request merged**